pkg/reconciler/broker/broker.go

```diff
@@ -469,3 +513,35 @@ func (r *Reconciler) httpsAddress(caCerts *string, b *eventingv1.Broker) pkgduck
 	httpsAddress.URL.Path = fmt.Sprintf("/%s/%s", b.Namespace, b.Name)
 	return httpsAddress
 }
+
+// getEventPolicyForBroker returns the EventPolicy for the Broker.
+func (r *Reconciler) getEventPolicyForBroker(b *eventingv1.Broker, chanMan *channelTemplate) (*eventingv1alpha1.EventPolicy, error) {
```

Comment:
As the lint check is indicating, it seems like we don't have anything in this that is leading to any errors, so maybe we don't need the error in the return type?

Reply:
done, PTAL!